scripts/auto_sshfs_mount.py

```diff
@@ -12,7 +12,7 @@
 
 #==============================================================================
 
-def Mount(local, remote, ServerAliveInterval=20, ServerAliveCountMax=3):
+def _Mount(local, remote, ServerAliveInterval=20, ServerAliveCountMax=3):
```

Comment:
Is there a particular reason for making this function and attempt_unmount semi-private? That sort of thing makes more sense in a class than a script.